src/app/auth/resetPassword/index.tsx

```diff
@@ -40,15 +46,13 @@ function ResetPasswordScreen() {
     }
   };
 
-  const checkPasswordMatchConfirmPassword = (text: string) => {
-    setPassword(text);
+  const checkPasswordMatchesConfirmPassword = (text: string) => {
     if (text !== '') {
       setIsMatching(text == confirmPassword);
     }
```

Comment:
I'm getting a bug where the update password button isn't getting disabled because when deleting a password on iOS, it deletes the entire field, making `text` = '', so this check doesn't run. Add an else statement that run `setIsMatching(false)` when the text is ''.